### ISSUE TITLE
Remote with ATV4K not working.. only for family members

### ISSUE THREAD

**Issue by rbressers** [nfarina/homebridge#1737]
Strangest thing.. homebridge works perfectly fine for me and my gf when we are in-house. We use an Apple TV 4K as a home hub. For me, homebridge works perfectly fine outside the house but my gf's phone shows "no response" for all homebridge accessories in the Home App and is not able to use them. The home hub is connected and HK supported devices (Elgato Eve & Hue) do work fine for both me and my gf outside the house.

Yes, I tried all suggestions in other related issues 
- Removed **everything** in the home app
- Resetted homebridge by changing usernames, PIN's and removing persistent directories
- Logged out of the ATV iCloud, restarted.. heck, I even resetted the damn thing to factory default
- factory resetted both iPhones
- removed all devices in homekit and added homebridge as only device

### Expected Behavior
When outside the house, homebridge accessories should be available on both my iPhone and my gf's iPhone because we use an ATV home hub.

### Current Behavior
For my iPhone (X, IOS11.2.5) it works fine but for my gf's iPhone (7, IOS11.2.5) homebridge accessories are not responding outside the house, using 4G/LTE. 

### Versions

- **Homebridge**: 0.4.33
- **Node**:       7.10.1
- **OS**:         RPi 3 on Raspbian stretch, kernel 4.9.79

### Configuration

```json
{
  "bridge": {
    "name": "Homebridge",
    "username": "11:22:33:44:55:66",
    "port": 51826,
    "pin": "xxx-xxx-xxx-xxx"
  },
  "platforms": [
    {
        "platform": "HomeAssistant",
        "name": "HomeAssistant",
        "host": "https://127.0.0.1:8123",
        "password": "xxxxxxxxxxxx",
        "supported_types": ["alarm_control_panel", "automation", "binary_sensor", "climate", "cover", "device_tracker", "fan", "group", "input_boolean", "light", "lock", "media_player", "remote", "scene", "script", "sensor", "switch", "vacuum"],
        "default_visibility": "hidden",
        "logging": true,
        "verify_ssl": false
    }
  ]
}
```

**Comment by zackaweed** [comment]
I have been having the exact same issue over the past week and have similarly tried everything you have to no avail. Hopeful for some thoughts.

**Comment by rbressers** [comment]
To be honest, I’m not ruling out the possibility that we’re hitting an Apple bug. I am experiencing the same results with Philips Hue when trying outdoors with a guest account on Homekit. Would be nice if anyone can debug the case to confirm as I am not sure.

**Comment by NorthernMan54** [comment]
I’m not sure how much help anyone here will be in figuring out this issue, but if it was myself and I was 100% sure that it wasn’t an issue on the HomeKit side ( highly likely ). I would start looking at my home network, and can my Apple TV see homebridge and are they talking.  If you run homebridge in debug mode, and you will see each device pair itself with homebridge.  If you don’t see the Apple TV, then are likely to have a network issue. Is your atv and rpi hard wired to the same network?

If your hue is not using homebridge then your issue is not homebridge.

Pro tip when trying to diagnose, turn off WiFi on your phone, and then your remote while still in the house.

**Comment by NorthernMan54** [comment]
Sorry read thru your details again, and there were issues in the late fall on the Apple side, where they turned off guest access, and patched a security issue.  For guest remote access to work, you must have the latest iOS and tvOS installed.  Older versions no longer work.

**Comment by rbressers** [comment]
All devices that are in use are using the latest available iOS releases 11.2.5 including tvOS. The issue doesn’t appear when only using Elgato Eve thermo’s but it breaks when adding homebridge.

**Comment by NorthernMan54** [comment]
And it works for your account remotely, and not for any guest? That is really odd.
